runtimes/eden/src/weights/pallet_utility.rs

```diff
@@ -78,10 +78,10 @@ impl<T: frame_system::Config> pallet_utility::WeightInfo for WeightInfo<T> {
 	// Proof Skipped: System Events (max_values: Some(1), max_size: None, mode: Measured)
 	/// The range of component `c` is `[0, 1000]`.
 	fn batch_all(c: u32, ) -> Weight {
-		// Minimum execution time: 15_511 nanoseconds.
-		Weight::from_parts(19_646_664_u64, 0)
-			// Standard Error: 1_867
-			.saturating_add(Weight::from_parts(9_687_695_u64, 0).saturating_mul(c as u64))
+		// Minimum execution time: 15_513 nanoseconds.
+		Weight::from_parts(37_100_643_u64, 0)
+			// Standard Error: 49_223
```

Comment:
I wonder if the higher standard error is related to the less "repeat" configuration. If it's then it may be worth repeating the benchmarks only for those pallets where the "Standard Error" reports are significantly higher.

Reply:
@aliXsed  you can revisit this one now.